doc/scandate-list-formats.rst

```diff
@@ -0,0 +1,129 @@
+Format of ``scandate`` list files
+=================================
+
+Many of the macros require a ``listOfScanDates.txt`` file. These come in either
+two or three column versions and :py:func:`anautilities.makeListOfScanDatesFile <gempython.gemplotting.utils.anautilities.makeListOfScanDatesFile>` is designed
+to parse either version and provide the tool with the correct information. This
+means that, baring other command line arguments, the two formats are relatively
+interchangeable.
+
+.. contents:: Table of Contents
+    :local:
+
+Two Column Format
+-----------------
+
```

Comment:
I know it does not appear in the `README.md` but here we may want to discuss how this is assuming the following format:

```
$DATA_PATH/<ChamberName>/<anaType>/<scandate>
```

Where `anaType` may be multiple subdirectories (e.g. for latency scans with tracking data `anaType = latency/trk`).  The function `getDirByAnaType` is used to determine this full path (from `$DATA_PATH` to the `scandate`).

Should probably include this before the two/three column format description since it applies to both.